lib/averages.js

```diff
@@ -0,0 +1,40 @@
+module.exports = Averages
+
+function Averages(array) {
+  this.numbers = array
+  this.arrayLength = array.length
+  this.total = this.numbers.reduce(function(sum, value) {
+      return sum + value;
+  }, 0);
+  this.mean = (this.total / this.arrayLength);
+  this.median = function() {
+    if (this.arrayLength % 2 == 0) {
+      var lowMid = this.numbers.sort(function(a, b){return a-b})[((this.arrayLength / 2) - 1)]
```

Comment:
sort can be a pretty heavy computation. you could save the sorted array as a variable then grab the `lowMid` and `highMid`